src/common/DeviceDescriptors.ts

```diff
@@ -1536,10 +1536,30 @@ export type DevicesMap = {
 };
 
 /**
- * @internal
+ * A list of devices to be used with `page.emulate(options)`. Actual list of devices can be found in {@link https://github.com/puppeteer/puppeteer/blob/main/src/common/DeviceDescriptors.ts | src/common/DeviceDescriptors.ts}.
+ *
+ * @example
+ *
+ * ```js
+ * const puppeteer = require('puppeteer');
+ * const iPhone = puppeteer.devices['iPhone 6'];
+ *
+ * (async () => {
+ *   const browser = await puppeteer.launch();
+ *   const page = await browser.newPage();
+ *   await page.emulate(iPhone);
+ *   await page.goto('https://www.google.com');
+ *   // other actions...
+ *   await browser.close();
+ * })();
+ * ```
+ *
+ * @public
  */
-export const _devicesMap: DevicesMap = {};
+const devices: DevicesMap = {};
```

Comment:
Should not only the type DevicesMap be public? I think this variable is only imported internally and then exposed via a getter,

Reply:
This is one of the exports that we are encouraging users to import directly into puppeteer. I.e.
```ts
import { devices } from 'puppeteer';
```
The getter on the main object has been deprecated.